src/main/flight/pid.c

```diff
@@ -515,7 +515,8 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
             previousRateError[axis] = rD;
 
             // if crash recovery is on and accelerometer enabled then check for a crash
-            if (pidProfile->crash_recovery && ARMING_FLAG(ARMED)) {
+            if (pidProfile->crash_recovery) {
+                if (ARMING_FLAG(ARMED)) {
                 if (motorMixRange >= 1.0f && inCrashRecoveryMode == false
```

Comment:
This still has the wrong indentation. It needs to be indented more than the surrounding block. As a consequence, your changes below have the wrong indentation too, confusing a casual reader of the code.